utils/upload_timestamps.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+# Copyright © 2020 Pavel Tisnovsky
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to retrieve timestamp of all objects stored in AWS S3 bucket and export them to CSV."""
+
+import boto3
+import botocore
+import csv
+import sys
+from argparse import ArgumentParser
+
+
+def connect_to_s3(aws_access_key_id, aws_secret_access_key, region_name):
+    """Try to connect into AWS S3 and initialize new session."""
+    session = boto3.session.Session(aws_access_key_id=aws_access_key_id,
+                                    aws_secret_access_key=aws_secret_access_key,
+                                    region_name=region_name)
+    use_ssl = True
+    endpoint_url = None
+    return session.resource('s3',
+                            config=botocore.client.Config(signature_version='s3v4'),
+                            use_ssl=use_ssl, endpoint_url=endpoint_url)
```

Comment:
Are `use_ssl` and `endpoint_url` always `True` and `None`?

Reply:
yup, that's expected there :)